x-pack/plugins/stack_connectors/server/connector_types/gemini/gemini.test.ts

```diff
@@ -0,0 +1,238 @@
+import { GeminiConnector } from './gemini';
```

Comment:
Some of these files are missing the license header, along with some violating basic eslint/prettier rules. You should be able to run the following command to automatically fix most of these issues:

```
node scripts/eslint x-pack/plugins/stack_connectors/ --fix
```

Reply:
fixed